programs/pkey/gen_key.c

```diff
@@ -249,6 +249,7 @@ int main(int argc, char *argv[])
     mbedtls_mpi_init(&DQ); mbedtls_mpi_init(&QP);
 #endif /* MBEDTLS_RSA_C */
 
```

Comment:
Please add a [changelog entry file](https://github.com/Mbed-TLS/mbedtls/blob/development/ChangeLog.d/00README.md) with an entry under “Bugfix”. See [ChangeLog](https://github.com/Mbed-TLS/mbedtls/blob/development/ChangeLog) for typical wording.